docs/policies.rst

```diff
@@ -393,5 +394,78 @@ It is recommended to use
           ``mu_neg = mu_pos`` and ``use_max_sim_neg = False``. See
           `starspace paper <https://arxiv.org/abs/1709.03856>`_ for details.
 
+Two Stage Fallback Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This policy handles low NLU confidence in multiple stages.
+
+- If a NLU prediction has a low confidence score, the user is asked to confirm
+  whether they really had this intent.
+
+    - If they confirm, the story continues as if the intent was recognized
+      with high confidence from the beginning.
+    - If they deny, the user is asked to restate his intent.
+
+- Clarification
+
+    - If the recognition of the clarification was confident, the story
```

Comment:
A bit unclear here.  Do you mean the `clarification intent`?

Reply:
there is no clarification intent. Any intent is accepted as clarification. 
Image a flow like

User: heeelooo <-- for example recognized as intent `bye` with low confidence
Bot: Did you mean goodbye?
user: no
Bot: Can you restate?
User: hello <-- clarification, correctly recognized as `greet`
Bot: Hi

Reply:
Any suggestion how to make this clearer? Is clarification a strange term for this?

Reply:
Clarification is fine, except it possibly leads to confusion when using `confirmation` in the same function.  You could try `reiteration`?

Reply:
Mhm, `reiteration` sounds strange to me. Something like `rephrasing` (is that a proper noun?) maybe? Would also change that in the code then.

Reply:
@MetcalfeTom

Reply:
`rephrasing` would be perfect